plugins/dev-tools/src/block_test_helpers.mocha.js

```diff
@@ -17,66 +17,81 @@ const {
 } = commonTestHelpers;
 
 /**
- * Code generation test case.
- * @extends {TestCase}
+ * Code generation test case configuration.
+ * @implements {TestCase}
  * @record
  */
-export function CodeGenerationTestCase() {}
-CodeGenerationTestCase.prototype = new TestCase();
-/**
- * @type {string} The expected code.
- */
-CodeGenerationTestCase.prototype.expectedCode = '';
-/**
- * @type {number|undefined} The expected inner order.
- */
-CodeGenerationTestCase.prototype.expectedInnerOrder = undefined;
-/**
- * @type {boolean|undefined} Whether to use workspaceToCode instead of
- * blockToCode for test.
- */
-CodeGenerationTestCase.prototype.useWorkspaceToCode = false;
-/**
- * A function that creates the block for the test.
- * @param {!Blockly.Workspace} workspace The workspace context for this test.
- * @return {!Blockly.Block}
- */
-CodeGenerationTestCase.prototype.createBlock = undefined;
+export class CodeGenerationTestCase {
+  /**
+   * Class for a code generation test case.
+   */
+  constructor() {
+    /**
+     * Creates the block to use for this test case.
+     * @param {!Blockly.Workspace} workspace The workspace context for this
+     *    test.
+     * @return {!Blockly.Block}
+     */
+    this.createBlock;
```

Comment:
I think methods are supposed to be declared in the body of the class, based on the linked guide. I think the only other place a function is declared is for `assertBlockStructure`.